src/views/Home.vue

```diff
@@ -3,6 +3,7 @@
     <div class="center">
       <b>{{ $i18n('welcome') }}</b>
       <router-link to="/story" class="create-story">{{ $i18n('btn-create') }}</router-link>
+      <router-link to="/viewer" class="view-story">{{ $i18n('btn-view') }}</router-link>
```

Comment:
Should we leave this button here as an temporary, easy way to access the viewer?

Reply:
Once the publish form is done, this can be removed.